src/sentry/api/event_search.py

```diff
@@ -353,7 +352,7 @@ class SearchFilter(NamedTuple):
     value: SearchValue
 
     def __str__(self):
-        return "".join(map(str, (self.key.name, self.operator, self.value.raw_value)))
+        return "".join(list(map(str, (self.key.name, self.operator, self.value.raw_value))))
```

Comment:
same here, `join(list(..))` can be `join(..)` afaik

Reply:
This is done in an optimization pass that I linked in the description. I guess we'd prefer it to be all rolled up in 1 PR though. https://github.com/getsentry/sentry/pull/28080#issuecomment-908578834